cluster/addons/pod-resizer/Makefile

```diff
@@ -0,0 +1,42 @@
+# The build rule builds a Docker image that resizes its Kubernetes pod as
+# the number of nodes in the cluster changes. The push rule pushes
+# the image to DockerHub.
+
+# Procedure for change:
+# 1. Bump the tag number.
+# 2. Create an image for the kubernetes user on Dockerhub and push there: make kbuild kpush
+# 3. Temporarily adjust any config files etc. that reference gcr.io/google_containers/fluentd-gcp:$(TAG)
+#    to use kubernetes/fluentd-gcp:$(TAG) instead.
+# 4. Test etc.
+# 5. Issue PR.
+# 6. When PR is approved make the gcr.io version of the image: make build push
+# 7. Revert the referendes to kubernetes/fluentd-gcp:$(TAG) to gcr.io/google_containers/fluentd-gcp:$(TAG) 
+
+OUT_DIR = build
+PACKAGE = k8s.io/contrib/pod-nanny
+PROJECT = google_containers
+TAG = 1.0
+
+# Rules for building the real image for deployment to gcr.io
+
+deps:
+		go get -u github.com/tools/godep
+
+compile: src/ deps
+		GOOS=linux GOARCH=amd64 CGO_ENABLED=0 godep go build -a -o $(OUT_DIR)/pod_nanny src/main/pod_nanny.go
+
+test: src/
+		godep go test ${PACKAGE}/src -v
+
+go: compile test
+
+clean:
+	rm -rf build
+
+build: go
+	docker build -t gcr.io/${PROJECT}/pod-resizer:$(TAG) .
```

Comment:
Some folks may not want to push to gcr.io -- please abstract the whole prefix out so that it's easy to push to dockerhub, quay, etc.